qdrant/v0.11.x/configuration.md

```diff
@@ -85,12 +86,12 @@ storage:
     indexing_threshold_kb: 20000
 
     # Interval between forced flushes.
-    flush_interval_sec: 1
+    flush_interval_sec: 5
     
-    # Max number of threads, which can be used for optimization. If 0 - `NUM_CPU - 1` will be used
-    max_optimization_threads: 0
+    # Max number of threads, which can be used for optimization.
```

Comment:
Since there is a difference in the meaning of the 0 value between `max_indexing_threads` (auto selection) and `max_optimization_threads`, I'd suggest putting a note:

_If 0 - optimization disabled._